developer_defaults.tf

```diff
@@ -145,6 +145,11 @@ variable "backup_retention_period" {
   default     = 0
 }
 
+variable "RETENTION_DAYS" {
```

Comment:
Try to maintain the naming conventions.
Maybe this should be `ebs_snapshot_retention_days`